src/components/desktop/Header.js

```diff
@@ -67,6 +68,10 @@ const StyledHeader = styled.header`
         font-weight: bold;
         font-size: 1rem;
 
+        @media (min-width: 767px) and (max-width: 820px) {
```

Comment:
I think you've correctly identified the reason for the layout to go funky, it's that min-width:110px that you're unsetting! I'm not sure if the solution is the best we can do, it happens to work in French, but at the cost of making the click area for the Map link quite small - see here with added background colors:
![Screenshot from 2024-11-16 15-50-53](https://github.com/user-attachments/assets/b55cd6ae-9d88-460b-8346-7662a2ba59c0)

Is there a way to have a flexible layout for these? Also, there'll be a third link there - the Activity tab - see #537 . If you're working on this part of the layout, you could actually add the button and the verbiage, let it not go anywhere for now, and I'll meanwhile help finish that PR.